modules/post/windows/gather/credentials/securecrt.rb

```diff
@@ -62,8 +62,13 @@ def try_encode_file(data)
   def enum_session_file(path)
     config_ini = []
     tbl = []
-    print_status("Searching for session files in #{path}")
-    config_ini += session.fs.file.search(path, '*.ini')
+    begin
+      print_status("Searching for session files in #{path}")
+      config_ini += session.fs.file.search(path, '*.ini')
+      fail_with(Failure::BadConfig, "Couldn't find any session files at #{path}") if config_ini == []
```

Comment:
totally subjective and likely not worth the effort changing but I think this would read better with `config_ini.empty?` that's just personal preference though

Reply:
Fair point I can update that.

Reply:
Applied fix with https://github.com/rapid7/metasploit-framework/pull/14341/commits/ef108eae4c953b197c7e2b16c15623cea2ec7c38

Reply:
`config_ini` is guaranteed to be an array right? I think you should just be able to do `config_ini.empty?` right? unless I'm missing something here
https://apidock.com/ruby/Array/empty%3F

Reply:
Its more of a personal safe guard since I've seen cases where we have assumed stuff would be an array but it isn't. Also you can't call `.empty?` on a `nil` element and since I've dealt with way too many stack traces related to lack of `nil` checks as of late, this tends to be my preferred go to solution.

Reply:
See convo below, this will be fixed in the next push.